### PR TITLE
Feat/log table

### PR DESCRIPTION
**Pull request by JIamep20**

### CODE REVIEW

**Comment by JIamep20** [comment]
@moshloop 
1. is "Create new hypothesis" button a replacement for "Link to incident" button?
image

2. Should this task include redesigning an existing modal or creating a new modal like in design?
image

File: src/components/Logs/Table/logs-table.js [hunk 1]
```diff
@@ -15,111 +22,164 @@ export function LogsTable({ logs, actions = [] }) {
...
+              type="button"
+              disabled={!Object.keys(selectedRowIds).length}
+              onClick={() => {
+                action.handler(selectedFlatRows.map((d) => d.original));
+              }}
+              className={clsx(
+                Object.keys(selectedRowIds).length > 0
+                  ? "text-indigo-700 bg-indigo-100 hover:bg-indigo-200"
+                  : "text-gray-400 bg-gray-200",
+                "inline-flex items-center px-2.5 py-1.5 border border-transparent text-xs font-medium rounded"
```
Comment (moshloop) [inline src/components/Logs/Table/logs-table.js:131]: please use btn-primary and btn-disabled classes

File: src/index.css [hunk 2]
```diff
@@ -19,5 +23,59 @@
   nav a.active {
     @apply bg-gray-100
   }
+
+  .standard-table {
```
Comment (moshloop) [inline src/index.css:27]: rename to comfortable-table + add styles for a compact table (with a button group selector for switching them)

File: tailwind.config.js [hunk 3]
```diff
@@ -52,7 +52,10 @@ module.exports = {
         "gray-color": "#C4C4C4",
         "dark-gray": "#242423",
         "medium-gray": "#6B7280",
+        "gray-warmer": "#A4A4A4",
```
Comment (moshloop) [inline tailwind.config.js:55]: Why do we have so many colors? e.g. gray-medium and medium-gray?  What is wrong with gray-300 and gray-900 ?

File: src/index.css (hunk 2 above)
Comment (JIamep20) [reply src/index.css:27]: image
1. is this "comptact table"?
2. is this button group selector for demo purposes, or am i missing something?

File: tailwind.config.js (hunk 3 above)
Comment (JIamep20) [reply tailwind.config.js]: removed new colors

File: src/index.css (hunk 2 above)
Comment (JIamep20) [reply src/index.css]: done

File: src/components/Logs/Table/logs-table.js (hunk 1 above)
Comment (JIamep20) [reply src/components/Logs/Table/logs-table.js]: removed this block of code, but used these classes in another place

File: src/components/Logs/Table/logs-table.js [hunk 4]
```diff
@@ -15,111 +22,164 @@ export function LogsTable({ logs, actions = [] }) {
...
+      {
+        id: "selection",
+        Header: "Time",
+        accessor: "timestamp",
+        Cell: function timestampCell({ cell: { row } }) {
+          return (
+            <div className="min-w-max flex flex-row">
+              <div className="mr-1.5">
+                <IndeterminateCheckbox
+                  className="focus:ring-indigo-400 h-4 w-4 text-indigo-600 border-gray-300 rounded"
```
[the remark this answers is not in the document]
Comment (JIamep20) [reply src/components/Logs/Table/logs-table.js]: implemented "checkbox" class and used it as default here

**Pull request merged**